### PR TITLE
Simplify

### PR DESCRIPTION
**Pull request by fregante**
I split the changes in multiple commits so let me know if something's wrong. Maybe the last commit is a bit too terse :P

### CODE REVIEW

File: content.js [hunk 1]
```diff
@@ -5,24 +5,15 @@ var rprotocol = /^https?:\/\//;
...
 
 function clicked (e) {
-  var a = e.target;
-  while (a) {
-    if (a.tagName === 'A') {
-      expand(a);
-      return;
-    }
-    a = a.parentElement;
+  var a = e.target.closest('a');
```
Comment (bevacqua) [inline content.js:8]: Can't believe I didn't know this API existed. Thanks!

File: content.js [hunk 2]
```diff
@@ -5,24 +5,15 @@ var rprotocol = /^https?:\/\//;
...
+  if (!a) {
+    return;
   }
-}
-
-function expand (a) {
-  var expanded = a.getAttribute('data-expanded-url');
-  if (expanded) {
-    a.href = expanded;
+  if (a.dataset.expandedUrl) {
```
Comment (bevacqua) [inline content.js]: Great changes, but let's add the variables back because minification/speed.

File: content.js (hunk 1 above)
Comment (fregante) [reply content.js:8]: I know, right?

File: content.js (hunk 2 above)
Comment (fregante) [reply content.js]: Done, although I'm not sure of what you mean by minification. Also... speed of variable creation vs property access, both quite fast :D

Comment (fregante) [reply content.js]: Also removed "merge functions" because my next PR will need that (adds support for Twitter cards, if I get that fully working)

**Pull request merged**